.github/workflows/oxfmt-ci.yml

```diff
@@ -0,0 +1,207 @@
+        with:
+          token: ${{ secrets.OXC_BOT_PAT }}
+          repository: oxc-project/oxc
+          issue-number: ${{ inputs.issue-number }}
+          comment-id: ${{ inputs.comment-id }}
+          reactions: eyes
+
+  build:
+    name: Build oxfmt
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+        with:
+          repository: oxc-project/oxc
+          ref: ${{ inputs.ref }}
+          persist-credentials: false
+          show-progress: false
+
+      - uses: oxc-project/setup-rust@1ff88fdaffd6ae35c0fb32ccc159340b37d0beac # v1.0.7
+        with:
+          save-cache: ${{ github.ref_name == 'main' }}
+
+      - uses: oxc-project/setup-node@fdbf0dfd334c4e6d56ceeb77d91c76339c2a0885 # v1.0.4
+
+      - uses: pnpm/action-setup@41ff72655975bd51cab0327fa583b6e92b6d3061 # v4
+
+      - run: pnpm install
+
+      - run: pnpm --filter ./apps/oxfmt run build
+        env:
+          RUSTFLAGS: "-C debug-assertions=true"
+
+      - run: pnpm --filter ./apps/oxfmt pack --pack-destination .
+
+      - uses: actions/upload-artifact@ea165f8d65b6e75b540449e92b4886f43607fa02 # v4
+        with:
+          if-no-files-found: error
+          name: oxfmt
+          path: ./oxfmt-*.tgz
+
+  checkout:
+    name: Read oxfmt-matrix.json
+    needs: build
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.setmatrix.outputs.content }}
+    steps:
+      - uses: taiki-e/checkout-action@b13d20b7cda4e2f325ef19895128f7ff735c0b3d # v1.3.1
+
+      - id: setmatrix
+        uses: jaywcjlove/github-action-read-file@d7632d78b76cbfbe8841dc31dd08dd2020175c80 # main
+        with:
+          localfile: ./oxfmt-matrix.json
+
+  test:
+    needs: checkout
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include: ${{ fromJson(needs.checkout.outputs.matrix) }}
+    name: Test ${{ matrix.repository }}
+    steps:
+      - name: Clone ${{ matrix.repository }}
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5
+        with:
+          repository: ${{ matrix.repository }}
+          ref: ${{ matrix.ref }}
+          path: ${{ matrix.path }}
+          persist-credentials: false
+          show-progress: false
+
+      - uses: actions/download-artifact@634f93cb2916e3fdff6788551b99b062d0335ce0 # v5
+        with:
+          name: oxfmt
+          path: ${{ matrix.path }}
+
+      - uses: actions/setup-node@2028fbc5c25fe9cf00d9f06a71cc4710d4507903 # v6
+        with:
+          node-version: 22
+
+      - uses: pnpm/action-setup@41ff72655975bd51cab0327fa583b6e92b6d3061 # v4
+        with:
+          version: 10
+
+      - name: Install oxfmt package
+        working-directory: ${{ matrix.path }}
+        run: |
+          # Extract and install the package
+          tar -xzf ./oxfmt-*.tgz
+          npm install -g ./package
```

Comment:
@Dunqing Could you please try installing `prettier` and `@prettier/sync` here for now?

Bundling them was harder than I expected... https://github.com/oxc-project/oxc/issues/15256